cvat-ui/src/containers/annotation-page/standard-workspace/objects-side-bar/object-item.tsx

```diff
@@ -404,6 +406,53 @@ class ObjectItemContainer extends React.PureComponent<Props> {
         }
     }
 
+    private switchCuboidOrientation = (): void => {
+        function orientation(points: number[]): boolean {
+            return points[12] > points[0];
+        }
+
+        const {objectState} = this.props;
+
+        this.resetCuboidPerspective(true);
+
+        objectState.points = shift(objectState.points,
+            orientation(objectState.points) ? 4: -4);
+
+        this.commit();
+    }
+
+    private resetCuboidPerspective = (notCommit?: boolean): void => {
+        function orientation(points: number[]): boolean {
```

Comment:
uninformative name of the function

Reply:
@bsekachev, what do you mean? Can you explain with more details?

Reply:
``if (orientation(points))`` is not very intuitive. What does this function do?